docs/check-examples

```diff
@@ -200,6 +200,7 @@ do
         capture=1
         dOptions=$defOptions
         if [ "$command" == "%check-verify" ]; then
+          ##cat $text
```

Comment:
Is this a change you meant to include in this PR?

Reply:
It is a debugging line -- I'll take it out in the next PR of this sort.